net_connection/src/json_protocol.rs

```diff
@@ -223,44 +335,75 @@ pub enum JsonProtocol {
     HandleSendMessageResult(MessageData),
 
     /// Request data from the dht network
-    #[serde(rename = "getDht")]
-    GetDhtData(GetDhtData),
```

Comment:
if we're changing all these to be suffixed with `Data` on the rust side, should we make the json rename match as well?